### ISSUE TITLE
[iOS] Move Course menu from bottom to top

### ISSUE THREAD

**Issue by sergeymomot** [openedx/openedx-app-ios#154]
Need to move Course menu from the bottom of the screen to the top of the screen. Course menu should scroll left and right. Course elements (Home-Videos-Dates etc.) should scroll left and right by swiping.

**Comment by marcotuts** [comment]
@sdaitzman - this is the ticket that could use design definition asap

**Comment by miankhalid** [comment]
@marcotuts @sdaitzman btw PR #181 does include a video showing the current implementation that you can review here: https://github.com/openedx/openedx-app-ios/pull/181#issuecomment-1833336934

**Comment by sdaitzman** [comment]
@miankhalid thanks, working to get design updates ready to share as soon as possible. The new top navigation has some key design improvements and looks great, but there are a couple changes we're likely to request:

- Including icons in each item, to make them easier to distinguish at a glance
- Potentially a different pill-shaped design with wider spacing, which would make the tap targets more distinct, and make the selected tab more clear

Here's an early mockup of these updates (pending some changes, but sharing this for reference). Do these changes seem feasible?

<img width="487" alt="image" src="https://github.com/openedx/openedx-app-ios/assets/1499476/eb0b4a30-a4f3-4bd2-b30e-73ca0242319a">

![image](https://github.com/openedx/openedx-app-ios/assets/1499476/84877d48-2867-432c-a0a8-a980778160af)

**Comment by sdaitzman** [comment]
Hi @ekangedx, thank you for raising these concerns. I want to make sure we address them fully and meet the usability heuristics you mentioned. I hope we can discuss them at the mobile design weekly tomorrow, and we can share out our findings here.

I think some of the usability concerns you raised have been addressed within our design process. For example, we tested the pill design and layout we propose on all common mobile device sizes from iPhone 12 Mini to iPhone 15 Pro Max, and the pill layout affords visual continuity on that full range. Happy to share more detail on the other concerns you raised, if that would be helpful.
